src/database/seeds/005_paciente.js

```diff
@@ -0,0 +1,32 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('paciente').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('paciente').insert([
+        {
+          id_usuario: 4,
+          peso: "65",
+          altura: "1.65",
+          alergia: "",
+          doenca_cronica: "",
+          vicio: "Cigarro",
+          medicamento: "",
+          created_at: null, 
+          updated_at: null 
```

Comment:
Deletar estas linhas caso você concorde em retirar esses atributos da tabela Paciente.
```suggestion
          medicamento: ""
```